### PR TITLE
Edit presets continued

### PR DESCRIPTION
**Pull request by Markvanderuit**
- Fixed toaster
- Add-preset dialog now has full functionality, and is styled a bit.
- Styled timeline toast differently.
- Small change to style of shotcaller buttons.

### CODE REVIEW

File: elements/timeline-view/timeline-dialog.html [hunk 1]
```diff
@@ -28,27 +32,36 @@
...
                         <paper-item id="{{item.id}}" preset={{item}} index="{{index}}">
+                            <paper-ripple></paper-ripple>
                             {{item.name}}
                         </paper-item>
                     </template>
                 </paper-menu>
             </paper-dialog-scrollable>
-            <p id="presetView" hidden>
+
+            <div id="presets_hide">
```
Comment (bart-de-jonge) [inline elements/timeline-view/timeline-dialog.html:48]: Please give a more descriptive name like no_presets_available

File: elements/timeline-view/timeline-dialog.html [hunk 2]
```diff
@@ -28,27 +32,36 @@
                 on-response="handleSetPresetsResponse"></iron-ajax>
 
         <paper-dialog id="dialog" with-backdrop>
-            <h2>Title goes here.</h2>
+            <h2>Please select a preset</h2>
 
-            <p>Header text goes here.</p>
-
-            <paper-dialog-scrollable>
+            <paper-dialog-scrollable id="presets_show">
```
Comment (bart-de-jonge) [inline elements/timeline-view/timeline-dialog.html:37]: More descriptive name like presets_available

**Comment by bart-de-jonge** [comment]
2 small comments, other than that great work!

File: elements/timeline-view/timeline-dialog.html (hunk 2 above)
Comment (Markvanderuit) [inline elements/timeline-view/timeline-dialog.html:37]: There's more places where I used that naming convention. Gimme a minute to run through the rest of my code.

File: elements/timeline-view/timeline-toast.html [hunk 3]
```diff
@@ -152,6 +221,33 @@
...
+
+        /**
+         * Respond to remove preset event
+         */
+        handleRemovePresetResponse(event) {
+            if(event.detail.response.success) {
+                this.$.toast.hide();
+                this.$.toast.show();
+
+                // just bloody force it
```
Comment (geenen124) [inline elements/timeline-view/timeline-toast.html:243]: lol

File: elements/timeline-view/timeline-toast.html [hunk 4]
```diff
@@ -137,13 +203,16 @@
             this.$.getPresets.url = "/benine/cameras/" + this.camera.id + "/presets";
             this.$.getPresets.generateRequest();
             this.$.toast.open();
+
+            this.$.instruments_available.hidden = this.block_data.instruments.size <= 0;
+            this.$.instruments_unavailable.hidden = this.block_data.instruments.size > 0;
```
Comment (geenen124) [inline elements/timeline-view/timeline-toast.html]: These 2 inline conditions can be extracted to a single boolean. Helps readability (e.g. `a.hidden = hideAvailableInstruments; b.hidden = !hideAvailableInstruments;`)

File: elements/timeline-view/timeline-dialog.html [hunk 5]
```diff
@@ -90,6 +103,23 @@
                 this.preset = activePreset;
                 this.$.presetView.hidden = false;
             }
+            this.$.presets_show.hidden = this.presets.length == 0;
+            this.$.presets_hide.hidden = this.presets.length > 0;
+
+            this.$.cancel_button.hidden = this.presets.length == 0;
+            this.$.accept_button.hidden = this.presets.length == 0;
+            this.$.close_button.hidden = this.presets.length > 0;
```
Comment (geenen124) [inline elements/timeline-view/timeline-dialog.html]: These inline conditionals are repeated. Please consider extracting them to local `const`s.

File: elements/timeline-view/timeline-toast.html (hunk 4 above)
Comment (Markvanderuit) [inline elements/timeline-view/timeline-toast.html]: Sure. There's plenty of places I did this, so I'll run through my changes and change it where I can.

**Comment by geenen124** [comment]
Great work! Will merge once the build passes

**Pull request merged**